docs/source/_gallery_original/index.rst

```diff
@@ -0,0 +1,64 @@
+AI Gallery
+====================
+
+
+AI Gallery is a collection of ready-to-run recipes for popular AI frameworks and AI models.
+It provides a simple way to **package**, **share**, and **distribute** AI projects using the simple interface of SkyPilot.
+
+Readers can directly execute the SkyPilot YAMLs on their own infrastructure, such as cloud VMs or Kubernetes.
```

Comment:
```suggestion
It provides a simple way to **package**, **share**, and **run** AI projects using the simple interface of SkyPilot.

Readers can directly execute the recipes on their own infrastructure, such as cloud VMs (link to install) or Kubernetes (link to k8s page).
```
Does the distribute -> run change look good? "distribute" is covered by "share".

Reply:
Sounds good! Updated.